src/Pages/youngbinkwon/Login/Login.js

```diff
@@ -1,13 +1,142 @@
-import React from './node_modules/react';
-import './Login.scss';
+import React, { Component } from "react";
+import { withRouter } from "react-router-dom";
+import "./Login.scss";
 
-class LoginKwon extends React.Component {
-    render() {
-        return(
-            <div className="LoginKwon">
-            </div>
-        )
-    }
+class Login extends Component {
+  goMain = () => {
+    this.props.history.push("/main-kwon");
+  };
+
+  render() {
+    return (
+      <div className="Login">
+        <LoginImage />
+        <div className="LoginPanel">
+          <LoginPanel goMain={this.goMain} />
+          <LoginPanelCreateAccount />
+          <LoginPanelGetApp />
+        </div>
+      </div>
+    );
+  }
+}
+
+class LoginImage extends Component {
+  constructor() {
+    super();
+    this.state = {
+      loginImage: [
+        { alt: "loginSlideImage1", src: "/img/youngbinkwon/loginimage1.jpg" },
+        { alt: "loginSlideImage2", src: "/img/youngbinkwon/loginimage2.jpg" },
+        { alt: "loginSlideImage3", src: "/img/youngbinkwon/loginimage3.jpg" },
+      ],
+    };
+  }
+
+  render() {
+    const { loginImage } = this.state;
+    return (
+      <div className="LoginImage">
+        {loginImage.map((img) => {
+          return <img alt={img.alt} src={img.src} />;
+        })}
+      </div>
+    );
+  }
+}
+
+class LoginPanel extends Component {
+  constructor() {
+    super();
+    this.state = {
+      userId: "",
+      userPassword: "",
+      btnColor: true,
+    };
+  }
+
+  //this.state이 한 단어 늦게 setstate되는 현상으로 인해 콜백함수 추가
+  handleInput = (e) => {
+    const { value, name } = e.target;
+    this.setState(
+      {
+        [name]: value,
+      },
+      this.validateAccount
+    );
+  };
+
+  validateAccount = () => {
+    const { userId, userPassword } = this.state;
+    const isValid = userId.includes("@") && userPassword.length >= 5;
+    this.setState({
+      btnColor: !isValid ? true : false,
```

Comment:
다시 보니까 이렇게도 줄일 수 있겠네요!
```suggestion
      btnColor: isValid
```
그리고 변수 명만 조금 바꾸면 
```suggestion
  const btnColor = userId.includes("@") && userPassword.length >= 5;
  this.setState({
   btnColor,
  })
      
```
이렇게도 줄일 수 있습니다!